embassy-usb/src/builder.rs

```diff
@@ -117,9 +120,9 @@ impl<'a> Config<'a> {
     /// Create default configuration with the provided vid and pid values.
     pub const fn new(vid: u16, pid: u16) -> Self {
         Self {
-            device_class: 0x00,
-            device_sub_class: 0x00,
-            device_protocol: 0x00,
+            device_class: 0xEF,
+            device_sub_class: 0x02,
+            device_protocol: 0x01,
```

Comment:
these need updating the default in the docs as well.

Reply:
Done